lib/ex_huobi/futures/websocket/market_ws.ex

```diff
@@ -49,6 +50,12 @@ defmodule ExHuobi.Future.Websocket.MarketWs do
         send(server, {:ws_reply, {:text, json}})
       end
 
+      defp pong_multiple(server, ts) do
+        send_after(server, {:pong, ts}, 1_000)
+        send_after(server, {:pong, ts}, 2_000)
+        send_after(server, {:pong, ts}, 3_000)
```

Comment:
can you explain the motivation of pong 3 times

Reply:
I guess websocket got disconnect because our pong sometime doesn't reach back to exchange due to network (error disconnect message on Timber is exactly like the one without pong in local), so I go for this solution.